python_modules/libraries/dagster-databricks/dagster_databricks/pipes.py

```diff
@@ -336,7 +336,7 @@ def name(self) -> str:
     # job has finished.
     def _get_log_path(self, params: PipesParams) -> Optional[str]:
         if self.log_path is None:
-            cluster_driver_log_root = params["extras"].get("cluster_driver_log_root")
+            cluster_driver_log_root = params.get("extras", {}).get("cluster_driver_log_root")
```

Comment:
should we have a different more explicit error when `extras` is not available instead?